### PR TITLE
Fixes SYNPY-927

### PR DESCRIPTION
**Pull request by kdaily**

### CODE REVIEW

File: synapseclient/__main__.py [hunk 1]
```diff
@@ -309,10 +309,15 @@ def delete(args, syn):
 def create(args, syn):
     _descriptionFile_arg_check(args)
 
-    entity = {'name': args.name,
-              'parentId': args.parentid,
-              'concreteType': 'org.sagebionetworks.repo.model.%s' % args.type}
-    entity = syn.createEntity(entity)
+    if args.parentid is None:
```
Comment (zimingd) [inline synapseclient/__main__.py]: deduplicate map creation

Can't do multi line suggestions:
```
entity = {'name': args.name,
          'concreteType': 'org.sagebionetworks.repo.model.%s' % args.type}
if args.parentid:
    entity['parentId'] = args.parentid
```

File: tests/integration/test_command_line_client.py [hunk 2]
```diff
@@ -981,3 +981,32 @@ def test_add__update_description():
...
+
+def test_create__same_project_name():
+    """Test creating project that already exists returns the existing project.
+
+    """
+
+    name = str(uuid.uuid4())
+    output_first = run('synapse',
+                       'create',
+                       '-name',
```
Comment (zimingd) [inline tests/integration/test_command_line_client.py]: per SYNPY-737
```suggestion
                       '--name',
```

File: tests/integration/test_command_line_client.py [hunk 3]
```diff
@@ -981,3 +981,32 @@ def test_add__update_description():
...
+                       'Project')
+
+    entity_id_first = parse(r'Created entity:\s+(syn\d+)\s+',
+                            output_first)
+
+    schedule_for_cleanup(entity_id_first)
+
+    output_second = run('synapse',
+                        'create',
+                        '-name',
```
Comment (zimingd) [inline tests/integration/test_command_line_client.py]: per SYNPY-737
```suggestion
                        '--name',
```

**Comment by kdaily** [comment]
Thanks @zimingd, fixed!

**Pull request merged**